samples/js/js-doc-snippets/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "js-doc-snippets",
+  "version": "0.0.1",
+  "type": "module",
+  "engines": {
+    "node": ">=14.0.0",
+    "npm": ">=6.0.0"
+  },
+  "dependencies": {
+    "@lightbend/akkaserverless-javascript-sdk": "0.30.0"
+  },
+  "devDependencies": {
+    "@lightbend/akkasls-scripts": "0.30.0",
```

Comment:
These could also be latest version `0.33.1`. There should probably be an update all the samples after release, like for Java SDK.

Reply:
made it the same as other samples `0.33.0` for now